stdlib/public/runtime/CrashReporter.cpp

```diff
@@ -33,3 +33,10 @@ struct crashreporter_annotations_t gCRAnnotations __attribute__((
 }
 
 #endif
+
+#if defined(_WIN32)
```

Comment:
Is the redeclaration necessary given `#include "swift/Runtime/Debug.h"`?

Reply:
Yes, note that the one in `Debug.h` is extern and this here is the real definition / storage location.